Dockerfile

```diff
@@ -33,7 +33,7 @@ RUN PIPENV_PIPFILE=tests/Pipfile pipenv install --system --deploy
 
 COPY Makefile ./
 COPY ./src/ ./src
-RUN make clean && make install -j8
+RUN make -s clean && make -s install -j8
```

Comment:
not really relevant but ok

Reply:
Yeah, I grew tired of seeing all the output when we just want to know that we had zero warnings, which `make -s` implements. I was not sure of adding that to the current PR but though same as you did, that it should be “ok”.